ls1intum/Apollon
# `Development`: Fix alignment of elements to grid issue

### Checklist


### Motivation and Context
This PR fixes the issue of elements not aligning perfectly on the grid.

**Changes include:**
- Set the default scale factor of Apollon to 0.8 instead of 0.75
- Set the moving/resizing factor of elements to 10 (previous value: 20)
- Fix alignment issue with following elements:
1. Rectangle in syntax tree 
2. Fork nodes
3. Terminal, Process and Function Call in Flow chart

### Description
Apollon only supports scaling factors with numbers having one decimal place (0.7, 0.8, 1,...)

### Steps for Testing
1. Clone the repo
2. Checkout to branch `enhancement/scaling-fix`
3. Execute commands: `yarn install`, followed by `yarn start`
4. Observe all the elements (especially the one mentioned above) are aligned perfectly with the grid.

### Screenshots
#### Before
![image](https://user-images.githubusercontent.com/14681902/204931063-27d85340-bc35-4898-b6ab-a7ea03638146.png)

#### After
![image](https://user-images.githubusercontent.com/14681902/204930662-d9abd526-aa84-4e44-8ab9-700ccc9bec97.png)
